examples/envoy/proxy.yaml

```diff
@@ -59,6 +59,7 @@ static_resources:
                         grpc_service:
                           envoy_grpc:
                             cluster_name: ratelimit
+                         transport_api_version: V3
```

Comment:
The indent here isn't right (one too many spaces) and it fails YAML parse. Do we have CI tests for this file?